### PR TITLE
EUI 8000 case file view fix

### PR DESCRIPTION
**Pull request by deanbullock-hmcts**
**Before creating a pull request make sure that:**

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-8000

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    fixture = TestBed.createComponent(CaseFileViewFieldReadComponent);
+    component = fixture.componentInstance;
+  });
+
+  it('should set allowMoving to true based on acl update being true', () => {
+    component.caseField = new CaseField();
+    component.caseField.acls = mockCaseFieldTrue.acls;
+
+    fixture.detectChanges();
+    expect(component.allowMoving).toBeTruthy()
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts:102]: @deanbullock-hmcts - missing semicolon

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  });
+
+  it('should set allowMoving to false based on acl update being false', () => {
+    component.caseField = new CaseField();
+    component.caseField.acls = mockCaseFieldFalse.acls;
+
+    fixture.detectChanges();
+    expect(component.allowMoving).toBeFalsy()
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts:110]: @deanbullock-hmcts - missing semicolon

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        role: 'caseworker-privatelaw-judge'
+      },
+      {
+        create: true,
+        read: true,
+        update: true,
+        delete: false,
+        role: 'caseworker-privatelaw-la'
+      },
+
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts:41]: @deanbullock-hmcts - can remove extra line

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        role: 'caseworker-privatelaw-judge'
+      },
+      {
+        create: true,
+        read: true,
+        update: false,
+        delete: false,
+        role: 'caseworker-privatelaw-la'
+      },
+
```
Comment (Daniel-Lam) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts:61]: @deanbullock-hmcts - can remove extra line

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@deanbullock-hmcts - some minor observations but the implementation itself is fine

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts (hunk 1 above)
Comment (LucaDelBuonoHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field-read.component.spec.ts:104]: We should have a rule in tslint.json for this

**Comment by deanbullock-hmcts** [comment]
> @deanbullock-hmcts - some minor observations but the implementation itself is fine

I still need to set up my formatter to auto tslint my code on save. Addressed. Thanks for pointing these out Dan :)

**Review by bradley-hmcts — DISMISSED** [review]

**Review by LucaDelBuonoHMCTS — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by Daniel-Lam — DISMISSED** [review]

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by Mo-Lala84 — DISMISSED** [review]
approved

**Review by RiteshHMCTS — DISMISSED** [review]

**Review by GaryScottHMCTS — DISMISSED** [review]

**Review by bradley-hmcts — APPROVED** [review]

**Review by RiteshHMCTS — APPROVED** [review]

**Pull request merged**